pandas/tests/indexes/categorical/test_constructors.py

```diff
@@ -136,12 +136,3 @@ def test_construction_with_categorical_dtype(self):
 
         with pytest.raises(ValueError, match=msg):
             Index(data, ordered=ordered, dtype=dtype)
-
-    def test_create_categorical(self):
```

Comment:
Does this no longer work? I think should keep the test no?

Reply:
the method it is testing is removed